jormungandr-lib/src/interfaces/vote.rs

```diff
@@ -101,13 +101,15 @@ impl<'de> Deserialize<'de> for SerdeMemberPublicKey {
                 formatter.write_str("binary data for member public key")
             }
 
+            #[allow(clippy::or_fun_call)]
             fn visit_bytes<E>(self, v: &[u8]) -> Result<Self::Value, E>
             where
                 E: serde::de::Error,
             {
-                Ok(SerdeMemberPublicKey(MemberPublicKey::from_bytes(v).ok_or(
-                    E::custom("Invalid binary data for member public key"),
-                )?))
+                let pk = MemberPublicKey::from_bytes(v).ok_or(serde::de::Error::custom(
```

Comment:
This should use `ok_or_else`, no need to construct a serde error on success. Don't override this lint.